### ISSUE TITLE
Uncaught TypeError: ref1.getModel is not a function

### ISSUE THREAD

**Issue by jsinai** [t9md/atom-cursor-history#29]
Tried to close the Mocha Test Runner pane.

**Atom**: 1.16.0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.4
**Thrown From**: [cursor-history](https://github.com/t9md/atom-cursor-history) package 0.9.1

### Stack Trace

Uncaught TypeError: ref1.getModel is not a function

```
At .atom/packages/cursor-history/lib/main.coffee:30

TypeError: ref1.getModel is not a function
    at closestTextEditor (/packages/cursor-history/lib/main.coffee:30:41)
    at /packages/cursor-history/lib/main.coffee:131:16
    at Function.module.exports.Emitter.simpleDispatch (/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:240:20)
    at CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:3:59)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/app.asar/src/command-registry.js:160:19)
    at /packages/mocha-test-runner/lib/result-view.coffee:25:47)
    at HTMLSpanElement.dispatch (/packages/mocha-test-runner/node_modules/jquery/dist/jquery.js:4435:9)
    at HTMLSpanElement.elemData.handle (/packages/mocha-test-runner/node_modules/jquery/dist/jquery.js:4121:28)
```

### Commands

```
```

### Non-Core Packages

```
atom-beautify 0.29.23 
atom-ternjs 0.18.2 
atom-typescript 11.0.2 
autoclose-html 0.23.0 
autocomplete-modules 1.6.9 
busy-signal 1.3.0 
cursor-history 0.9.1 
file-icons 2.1.3 
git-plus 7.6.0 
hyperclick 0.0.40 
intentions 1.1.2 
js-hyperclick 1.10.1 
language-brightscript 0.6.3 
linter 2.1.4 
linter-eslint 8.1.6 
linter-jshint 3.0.3 
linter-ui-default 1.2.3 
merge-conflicts 1.4.4 
mocha-test-runner 0.5.1 
node-debugger 1.10.1 
react 0.16.2 
vim-mode-plus 0.91.0 
```

**Comment by t9md** [comment]
Can you reproduce always?
How can I run reproduce( how can I run mocha test runner )?

**Comment by jsinai** [comment]
Yes, I can still reproduce it. Instructions:

- Unzip the attached
- Install the mocha-test-runner atom package
- Install node.js
- In the project directory, run npm install
- On the command line, start the server by running: node node-debugger.js
- In Atom, open test.js
- Type ctrl-alt-m (or equivalent on your platform). This should run the test
- The mocha results window should open automatically
- Click the X at the right of this window
- You should see the error
[Archive.zip](https://github.com/t9md/atom-cursor-history/files/1046663/Archive.zip)

**Comment by t9md** [comment]
Thanks, could reproduce by my side. Will investigate from now.

**Comment by t9md** [comment]
released as v0.11.1 👍

**Comment by jsinai** [comment]
It's fixed, thanks!
